trieste/acquisition/function/continuous_thompson_sampling.py

```diff
@@ -42,6 +42,16 @@ class GreedyContinuousThompsonSampling(SingleModelGreedyAcquisitionBuilder[HasTr
     :cite:`wilson2020efficiently`.
     """
 
+    def __init__(self, output_dim: int = 0):
```

Comment:
I think it would be useful to have a callable here, say `select_output` instead of `output_dim`, the default one could be a simple function that would choose a dimension, but it would allow for more complex combination of multiple outputs

Reply:
Ok interesting, so for instance you could take the sum of multiple outputs and optimize that instead?

Reply:
yes, or cover the example that @vpicheny  was giving, with a heteroscedastic model
"Take hetGP, if you want to do TS over the mean, you do TS over the first GP; but if you want to do TS over a quantile, you'd get a quantile trajectory by traj_gp_1 + beta * exp(traj_gp_2)]."